draft-birkholz-scitt-software-supply-chain-use-cases.md

```diff
@@ -107,18 +108,32 @@ The following steps are performed by the Software Producer:
 The SBOM must follow NTIA minimum elements for SBOM’s and other NTIA and NIST recommendations for SBOM, to meet Executive Order 14028 and OMB M-22-18 requirements.
 (Co)SWID, SPDX or CycloneDX SBOM formats are acceptable for this artifact. 
-2. Digitally sign the SBOM artifact.
-3. Place the SBOM and digital signature artifacts within an access-controlled location, i.e. a customer portal, and provide the end consumer with a link to these artifacts for downloading to the customers environment.
+2. The encoding of SBOM is specific to the protocol in question.
+3. Makes a claim to establish its own identity as a producer of the overall SBOM.
+4. The issuer in certain situations can make further claims (example claims that relate to steps that might assist independent building of the software components, to arrive at the final deliverable).
+5. Once all the claim construction is complete, make a CBOR data payload of all the claims.
+6. Signs the payload using SCITT recommended signing scheme. For now this will be COSE signing. 
+7. Submits the COSE signed object to the transparent registry.
+8. Receives the return receipts (A COSE Countersignature object, containing the inclusion proofs, the original COSE payload and a signature from the transparency service)
+9. Producer would then ship the software along with the return receipts to the Software Consumer.
 
 ### Consumer Actions
 
-A software consumer, in this case DOI, obtains a digitally signed SBOM artifact from a software vendor, which initiates the following risk assessment process:
+A software consumer, in this case DOI, receives the software deliverable along with return receipts from the Producer.
+
+1. The Consumer may choose to verify that the received software specific claims are indeed present in the Transparency Service.
+2. It will decode the return receipts to get the inclusion proof.
+3. Using inclusion proofs it will cyrpotgraphically evaluate the presence of Supply Chain Artifacts (including SBOM) in the Transparency Service.
+4. It will decode the CBOR Payload to get the SBOM and other claims made by the issuer.
+5. It may do additional verification to check that the identity claim value matches the identity of the issuer from where it received the software or the trust chain linking to the issuer.
+6. A format aware consumer can decode the SBOM contents and do the SBOM specific verification steps to further augment the trustworthiness of the received deliverable.
+7. The consumer can decode other claims to establish augmented trust on the received software. for example decode the specific claim to get the required instructions to build the software it has received. This way it can compare the computed hash for the binary compiled locally against the received hash from SCITT registry. Any mis-match can be reported as an inconsistency.
```

Comment:
```suggestion
7. The consumer can decode other claims to establish augmented trust on the received software. For example decode the specific claim to get the required instructions to build the software it has received. This way it can compare the computed hash for the binary compiled locally against the received hash from SCITT registry. Any mis-match can be reported as an inconsistency.
```